cinema_service/urls.py

```diff
@@ -4,5 +4,7 @@
 urlpatterns = [
     path("admin/", admin.site.urls),
     path("api/cinema/", include("cinema.urls", namespace="cinema")),
+    path("api/user/", include("user.urls"), name="user"),
```

Comment:
The `include` function should use `namespace="user"` instead of `name="user"`. Correct it to: `path("api/user/", include("user.urls", namespace="user"))`.